src/panels/SessionRecording/Buttons/PausePlaybackButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { Button, ButtonProps } from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { PauseIcon } from '@/icons/icons';
+
+export function PausePlaybackButton({ ...props }: ButtonProps) {
```

Comment:
How about naming these like
PauseButton
PlayButton
StopButton

?

Reply:
Can we unify them in terms of if they have their own onClick function or if it passed as a prop?